modules/olm-operatorhub.adoc

```diff
@@ -30,6 +30,11 @@ ISVs to package and ship. Supported by the ISV.
 |Optionally-visible software maintained by relevant representatives in the
 link:https://github.com/operator-framework/community-operators[operator-framework/community-operators]
 GitHub repository. No official support.
+
+|Custom Operators
+|Operators you add to the cluster yourself.
+If you have not added any Custom Operators, the Custom category will not appear in
```

Comment:
s/will not/does not

Reply:
Sure. Kathryn, can you  merge this once you are done reviewing it? I don't have permission to do any of that.

Reply:
Chris, if you make the changes and squash, I'm happy to merge it.

Reply:
Change done and squashed.